src/lib.rs

```diff
@@ -105,6 +106,10 @@ impl<'a> AtomicWriteBuffer<'a> {
             None
         }
     }
+
+    pub fn get_buffer(&self) -> &'a [u8] {
```

Comment:
Is this necessary?
→ Because 'a' is live.

Alternatively, how about the following?
```suggestion
    pub fn into_inner(self) -> &'a [u8] {
```